packages/veritone-react-common/src/components/OCREngineOutputView/index.js

```diff
@@ -0,0 +1,122 @@
+import React, { Component, Fragment } from 'react';
+import { arrayOf, shape, number, string, func } from 'prop-types';
+import classNames from 'classnames';
+
+import EngineOutputHeader from '../EngineOutputHeader';
+import withMuiThemeProvider from '../../helpers/withMuiThemeProvider';
+import OCRObject from './OCRObject';
+import styles from './styles.scss';
+
+@withMuiThemeProvider
+class OCREngineOutputView extends Component {
+  static propTypes = {
+    data: arrayOf(
+      shape({
+        startTimeMs: number,
+        endTimeMs: number,
+        sourceEngineId: string,
+        sourceEngineName: string,
+        taskId: string,
+        series: arrayOf(
+          shape({
+            startTimeMs: number,
+            stopTimeMs: number,
+            object: shape({
+              text: string
+            })
+          })
+        )
+      })
+    ),
+    engines: arrayOf(
+      shape({
+        sourceEngineId: string,
+        sourceEngineName: string
+      })
+    ),
+    selectedEngineId: string,
+    onEngineChange: func,
+    onExpandClicked: func,
+    onOcrClicked: func,
+    className: string,
+    currentMediaPlayerTime: number
+  };
+
+  static defaultProps = {
+    data: []
+  };
+
+  handleOcrClick = (startTime, stopTime) => evt => {
+    this.props.onOcrClicked && this.props.onOcrClicked(startTime, stopTime);
+  };
+
+  render() {
+    let {
+      data,
+      className,
```

Comment:
perhaps it would be useful to rename this to be more descriptive, className for what?